### PR TITLE
Remove old octreeFade code that wasn't working anyway

### PR DESCRIPTION
**Pull request by howard-stearns**
Per discussion in https://github.com/highfidelity/hifi/pull/5272, the octreeFade code was old, obsolete, counter to current practice (naked ogl), and now it doesn't even work properly (flashing a box on just part of the screen instead of all of it). And ugly, too.

This pr removes it.

### CODE REVIEW

File: tests/ui/src/main.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ class MenuConstants : public QObject{
         DisplayModelElementProxy,
         DisplayDebugTimingDetails,
         DontDoPrecisionPicking,
-        DontFadeOnOctreeServerChanges,
+        RESERVED_DontFadeOnOctreeServerChanges,
```
Comment (ZappoMan) [inline tests/ui/src/main.cpp]: I don't think this is needed. @jherico - can you confirm. I think this test code is yours.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 51be24515974e60fe4625c979d3644981d5383c0 available at: https://gist.github.com/f84490d3e59b7ebc4482<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5307.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5307.dmg

File: tests/ui/src/main.cpp (hunk 1 above)
Comment (jherico) [inline tests/ui/src/main.cpp]: yeah, that's junk.

Comment (howard-stearns) [inline tests/ui/src/main.cpp]: OK, thanks! I"ll remove it an update the pr within a few minutes.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5196648b229e6216e5043f3dbf5eaa50c02b4458 available at: https://gist.github.com/f84490d3e59b7ebc4482<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5307.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5307.dmg

**Pull request merged**